qutebrowser/completion/models/miscmodels.py

```diff
@@ -122,3 +122,22 @@ def delete_buffer(data):
         model.add_category(cat)
 
     return model
+
+
+def window(*, info=None):  # pylint: disable=unused-argument
```

Comment:
@rcorre can you take a quick look at this?

Reply:
LGTM, just needs a test in `test_models`